dataactbroker/handlers/accountHandler.py

```diff
@@ -479,7 +480,7 @@ def listUsersWithStatus(self):
             return JsonResponse.error(exc,exc.status)
         userInfo = []
         for user in users:
-            thisInfo = {"name":user.name, "title":user.title,  "agency":user.agency, "email":user.email, "id":user.user_id }
+            thisInfo = {"name":user.name, "title":user.title,  "cgac_code":user.cgac_code, "email":user.email, "id":user.user_id }
```

Comment:
Same question as above re: is it useful for the caller of this function to also get the agency name? Especially since that's what was being returned previously.